requirements_dev.txt

```diff
@@ -8,7 +8,7 @@ tabulate
 numpy>=1.9.1
 scipy>=0.9.0
 blessings>=1.5.1
-pandas>=0.15.2
+pandas==0.18.1
```

Comment:
I think this in only in there for tests to pass because this branch still contains the pandas pickles. Should be set to `pandas>=0.21.1` eventually